webview2-sys/idl2rs/src/idl.pest

```diff
@@ -27,7 +27,8 @@ method_name = { identifier }
 method = { doc_comment? ~ method_attribute? ~ _type ~ method_name ~ "(" ~ parameters ~ ")" ~ ";" }
 
 // Enum.
-variant = { doc_comment? ~ identifier }
+variant_value = { ("0x" ~ (('A' .. 'F') | ('0' .. '9'))+) | digit+ }
+variant = { doc_comment? ~ identifier ~ (WHITESPACE? ~ "=" ~ WHITESPACE? ~ variant_value ~ WHITESPACE?)? }
```

Comment:
`WHITESPACE` shouldn't need to explicitly specified?

https://pest.rs/book/grammars/syntax.html#implicit-whitespace